examples/sample-pub/metering.js

```diff
@@ -44,6 +44,17 @@ const MeteringDemo = {
     GaaMeteringRegwall.signOut().then(() => void location.reload());
   },
 
+  /** Mocks registration of a user, given data from Google Sign-In. */
```

Comment:
Could you add a comment about what pubs should do in production?

Reply:
I updated the docs to suggest setting a 1p http-only cookie so it lives past 7 days in Safari